src/styl/main.styl

```diff
@@ -17,10 +17,81 @@
 * Highlight Themes
 * uncomment your favorite theme
 * check them live at: https://highlightjs.org/static/demo/
-* and convert them from this: https://github.com/isagalaev/highlight.js/tree/master/src/styles 
+* and convert them from this: https://github.com/isagalaev/highlight.js/tree/master/src/styles
 */
-@import "highlight-themes/zenburn"
-
+@import "highlight-themes/agate"
+// @import "highlight-themes/androidstudio"
```

Comment:
Please, remove all those commented codes and just leave a comment saying the folder with another highlight-themes. I thinks its better now, because we have many themes.